### PR TITLE
Update SelectionSort2.java

### PR DESCRIPTION
**Pull request by sergiovanovi**
Small method

### CODE REVIEW

File: 02_selection_sort/java/src/SelectionSort2.java [hunk 1]
```diff
@@ -3,45 +3,23 @@
...
-        int smallestIndex = 0;
-        for (int i = 0; i < arr.length; i++) {
-            if (arr[i] < smallest) {
-                smallest = arr[i];
-                smallestIndex = i;
+    public static void selectionSort(int[] target) {
+        for (int i = 0; i < target.length - 1; i++) {
+            int left = target[i];
+            for (int j = i + 1; j < target.length; j++) {
+                int rigth = target[j];
```
Comment (egonSchiele) [inline 02_selection_sort/java/src/SelectionSort2.java:10]: please change `rigth` to `right` and I'll merge

**Review by egonSchiele — CHANGES_REQUESTED** [review]
spelling correction needed

File: 02_selection_sort/java/src/SelectionSort2.java (hunk 1 above)
Comment (sergiovanovi) [reply 02_selection_sort/java/src/SelectionSort2.java]: +

Comment (sergiovanovi) [reply 02_selection_sort/java/src/SelectionSort2.java]: Done

**Comment by sergiovanovi** [comment]
done

**Pull request merged**